recursive_metadata/README.md

```diff
@@ -0,0 +1 @@
+Utility scripts that pull/write all metadata associated with an urn and any references it may have.
```

Comment:
This needs a bit more info. Maybe a single example of what is input/ouput of scripts?